src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/renametool/ConfirmationModalPage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ConfirmationModalPage extends SpecialPageObject {
+
+  //TODO: Move it to components
+  @FindBy(css = "")
+  private WebElement messageTextBox;
+  @FindBy(css = "#WikiaConfirmOk")
+  private WebElement yesButton;
+  @FindBy(css = "#WikiaConfirmCancel")
+  private WebElement noButton;
+  By confirmationmodalContainerBy = By.cssSelector("");
+
+  public ConfirmationModalPage(WebDriver driver) {
+    super();
+    PageFactory.initElements(driver, this);
+  }
+
+  public boolean isVisible() {
+    return !driver.findElements(confirmationmodalContainerBy).isEmpty();
+  }
+
+  public String getMessage() {
+    return messageTextBox.getText();
+  }
+
+  public void confirm() {
+    yesButton.click();
```

Comment:
`clickConfirm`, `clickReject`, `clickAccept` would be more descriptive. Also can you add `wait.forElementClickable()` before each of these click actions?